cli/src/command/finalize.rs

```diff
@@ -114,13 +114,24 @@ impl Finalize {
             .consensus_key)
     }
 
-    fn create_connect_list_config(public_configs: &[NodePublicConfig]) -> ConnectListConfig {
-        // FIXME: filter self (ECR-4190).
+    fn create_connect_list_config(
+        public_configs: &[NodePublicConfig],
+        skipped_key: &PublicKey,
```

Comment:
nit: `skip_key` is more a bit more logical variable name IMO

Reply:
@vitvakatu `skip_key` is a verbal phrase though; IMO, noun phrases are more appropriate for variable names.

Reply:
`skipped_key` means "the key that was skipped", while the actual meaning is "the key that is needed to be skipped". As `key_to_skip` is a bit long, I suggested to use `skip_key`, which is used nicely in a function call like `create_connect_list_config(..., skip_key: key)`. However, I'm not gonna to argue about variables names, it was a nit.

Reply:
IMHO `key_to_skip` is just fine.